package.json

```diff
@@ -22,14 +22,15 @@
   "license": "Apache-2.0",
   "dependencies": {
     "lodash": "4.17.11",
-    "request": "2.88.0"
+    "request": "2.88.0",
+    "request-promise-native": "^1.0.7"
```

Comment:
Request module is a feature-full module but it's slower and heavy weight.  In your use case, you might be able to use https://www.npmjs.com/package/node-fetch instead.

Reply:
The `request` module was already there and being used by `partnerHelper.js`. It doesn't seem worthwhile to introduce `node-fetch` for this one call that's made during the token granting process, and I don't want to mess with partnerHelper as part of this PR and risk breaking something. 

I did add `request-promise-native`, which I can easily drop in favor of just `request` and creating a Promise myself for this one call, as partnerHelper does, but is that worth it? Looking at Viper I see that you are using `request` and then doing a `promisefy` from the `util` module...
```
const request = require("request");
const {promisify} = require("util");
let rp = promisify(request);
```
Is repeating that pattern preferable for some reason? I'm asking not so much in respect to this one token call that happens only on installation, but regarding what we want to adopt as our standard practices across NodeJS examples and SDKs.

Reply:
:thumbsup: I went ahead and switched the stuff I added to `node-fetch` and removed the dependency on `request-promise-native`. I've left `partnerHelper.js` alone for now, and so there's still a project dependency on `request`. It's not clear to me that partnerHelper is really intended for production use or that the request builder function it provides adds enough value to be worth maintaining. The command-to-state mapping it provides is incomplete as well. But that's a topic for future discussion.